files/default/sql/ddl/3.8.0__initial_tables.sql

```diff
@@ -2078,11 +2076,26 @@ CREATE TABLE IF NOT EXISTS `transformation_function` (
                                                          `feature_store_id`                  INT(11)         NOT NULL,
                                                          `created` timestamp NULL DEFAULT CURRENT_TIMESTAMP,
                                                          `creator` int(11) NOT NULL,
+                                                         `save_type`                         VARCHAR(255)    NOT NULL, 
                                                          PRIMARY KEY (`id`),
                                                          CONSTRAINT `feature_store_fn_fk` FOREIGN KEY (`feature_store_id`) REFERENCES `feature_store` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION,
                                                          CONSTRAINT `creator_fn_fk` FOREIGN KEY (`creator`) REFERENCES `users` (`uid`) ON DELETE NO ACTION ON UPDATE NO ACTION
 ) ENGINE = ndbcluster DEFAULT CHARSET = latin1 COLLATE = latin1_general_cs;
 
+--
+-- Table structure for table `feature_view_transformation_functions`
+--
+
+CREATE TABLE IF NOT EXISTS `feature_view_transformation_function` ( -- mapping transformation functions in feature view with required features and actual transformation function.
```

Comment:
will you migrate the existing feature view which are using built-it transformation functions to this table?

Reply:
Yes migration of existing feature views that using built-in transformations needs to be done.

Reply:
ok, will you add it here or in expat?

Reply:
To migrate I believe I would have to :
1. Copy the modified transformation function source code into the transformation function folder of each project that has already been created.
2. Add entires into the table `feature_view_transformations` for each feature view that already has transformation functions.

So where do you think it would be best to do the migration. Can I do it here or should it be in expat.

Reply:
I think you need to delete the old transformation function and attach the new transformation function for each feature view which have transformation function. You can do it in expat.

Reply:
Okey I will start working on that too.